PlanPluginBridge/pom.xml

```diff
@@ -167,6 +167,12 @@
             <version>1.6.0-PR1-S7.0</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
I'm guessing luckperms is provided in jitpack.io - Could you add that to the repository comment above in repository section.
If it is not available in a repository, please move the dependency below the `Plugins requiring local install` comment (They should be in alphabetical order)

Reply:
Maven central